### PR TITLE
Modify shengzhang09 caption file

### PR DESCRIPTION
**Pull request by shengzhang09**
from sheng

### CODE REVIEW

**Comment by mgullon** [comment]
What? 
:+1:

File: _posts/0000-01-13-shengzhang09.md [hunk 1]
```diff
@@ -6,4 +6,4 @@ title: "Welcome to our slide deck!"
 ![puss-in-boots](https://cloud.githubusercontent.com/assets/16547949/25401020/740dd682-29c2-11e7-9906-ac5c3a8496c7.jpg)
 {: .center}
 
-Caption here
+puss in boots
```
Comment (mgullon) [inline _posts/0000-01-13-shengzhang09.md]: interesting!

**Comment by qhui-red** [comment]
🕚 🕦 🕛 🕧 🕑 🕝 🕒 🕞 🕓 🕟 🕔 🕠 🕕 🕡 🕖 🕢 🕗 🕣 🕘 🕤

**Review by mgullon — APPROVED** [review]
ok

**Pull request merged**